libbpf-rs/src/program.rs

```diff
@@ -419,4 +432,8 @@ impl Program {
     pub fn insn_cnt(&self) -> usize {
         unsafe { libbpf_sys::bpf_program__insn_cnt(self.ptr) as usize }
     }
+
+    pub fn unload(&mut self) {
```

Comment:
Do we need to consume `self` here? Does closing the prog break any invariants held by `Program`?